aws/app/lambda/audit_logs/audit_logs.js

```diff
@@ -81,6 +81,8 @@ exports.handler = async function (event) {
         },
       })
     );
+    console.log("AuditLogs");
```

Comment:
Are the two `console.log` required for future debugging?

Reply:
Yes, I wanted to ensure exactly what's being returned from the dynamo DB to ensure the logic below was sound.